<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>521</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>RC Input</string>
    </attribute>
    <widget class="QLabel" name="ch0Max">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch2Rev">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="ch4Slider">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>18</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ch0Cur">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch1Min">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch7Rev">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ch6Min">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch7Cur">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch4Max">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch7Min">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getCurrent">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Current</string>
     </property>
    </widget>
    <widget class="QSlider" name="ch3Slider">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>18</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ch2Min">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch6Max">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch0Rev">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ch5Max">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch3Max">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch1Rev">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="saveToSD">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>210</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save to SD</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch4Cur">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch4Min">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QSlider" name="ch7Slider">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>18</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="saveToRAM">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>210</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save to RAM</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ch0Assign">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch5Rev">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="ch1Slider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>18</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch6Rev">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ch3Min">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch7Max">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch2Cur">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch5Cur">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
    </widget>
    <widget class="QSlider" name="ch2Slider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>18</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ch5Min">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch1Cur">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch6Cur">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch3Cur">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
    </widget>
    <widget class="QLabel" name="ch2Max">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QSlider" name="ch0Slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>18</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ch1Max">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch4Rev">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ch0Min">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QSlider" name="ch5Slider">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>18</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="ch6Slider">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>18</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="ch3Rev">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FreeSans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_8">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
